index.html

```diff
@@ -0,0 +1,236 @@
+          <div class="PrimaryText">
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
+          </div>
+          <div class="SupportingText">
+            <p class="SupportingText">I'm a Software Developer</p>
+          </div>
+          <div class="discription">
+            <p class="discription">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hestiate to contact me.
+            </p>
+          </div>
+          <ul class="SocialMedia">
+            <li>
+              <a href="https://github.com/Robison-faike123">
+                <img src="images/Icon -GitHub.png" alt="-GitHub" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/">
+                <img src="images/Icon - Linkedin.png" alt="Linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Angelist.png" alt="Angelist" />
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com">
+                <img src="images/Icon - Twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Medium.png" alt="Medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <!--this is for second section -->
+    <section class="Works">
+      <div class="RecentWorks">
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="MultiPostStories">
+        <img
+          class="MyResentWorkImg1"
+          src="/images/MyRecentWorkImg1.png"
+          alt=""
+        />
+    
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languageBoxes">
+          <li class="box">
+            <p>CSS</p>
+          </li>
+          <li class="box">
+            <p>HTML</p>
+          </li>
+          <li class="box">
+            <p>BOOTSTRAP</p>
+          </li>
+          <li class="box">
+            <p>RUBY</p>
+          </li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w1">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
```

Comment:
- [ ] Kindly add some margin/padding around the section that borders the technologies lists and the text description. This way, it will match the provided Figma design and meet all the project requirements.

**This is how I am seeing this section right now:**

![image](https://user-images.githubusercontent.com/86318284/192717473-72ef5e2e-c4fd-43e4-a03e-d9b5be1fac71.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/192717582-379b86c0-a701-4f60-b2ce-b92a95ef9c91.png)

**Kindly ensure you implement this on all the cards**

Reply:
it has been worked on